NotadogApi/Hubs/GameHubMethod.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NotadogApi.Hubs
+{
+	public enum GameHubMethod
```

Comment:
cool!

Reply:
Every time you use the same string literal think about enums or constants.